include/object/notification.h

```diff
@@ -36,4 +37,31 @@ static inline void maybeReturnSchedContext(notification_t *ntfnPtr, tcb_t *tcb)
 }
 #endif
 
+#ifdef CONFIG_KERNEL_MCS
```

Comment:
Need to add the `&& CONFIG_FASTPATH`

Reply:
Is this referring to the definition of `maybeDonateSchedulingContext`? Isn't this used for the slowpath too though? I just moved [this definition from `notification.c`](https://github.com/seL4/seL4/blob/master/src/object/notification.c#L41-L51) (where it didn't have `CONFIG_FASTPATH`) to `notification.h`.

Reply:
If you've just moved it, it shouldn't have `CONFIG_FASTPATH`, yes.